app/translations/index.js

```diff
@@ -6,8 +6,14 @@ import Translation from './src/Translation';
 import DummyTranslation from './src/DummyTranslation';
 import TranslationFragment from './src/TranslationFragment';
 import DateFormatter from './src/DateFormatter';
+import GetDeviceLocale from './src/GetDeviceLocale';
 
-export { DummyTranslation, TranslationFragment, DateFormatter };
+export {
+  DummyTranslation,
+  TranslationFragment,
+  DateFormatter,
+  GetDeviceLocale,
```

Comment:
Is it necessary to make it public? I would prefer to keep it private as long as possible...